easybuild/easyconfigs/g/GDAL/GDAL-3.3.0-foss-2021a.eb

```
@@ -0,0 +1,61 @@
+easyblock = 'ConfigureMake'
+
+name = 'GDAL'
+version = '3.3.0'
+
+homepage = 'https://www.gdal.org'
+description = """GDAL is a translator library for raster geospatial data formats that is released under an X/MIT style
+ Open Source license by the Open Source Geospatial Foundation. As a library, it presents a single abstract data model
+ to the calling application for all supported formats. It also comes with a variety of useful commandline utilities for
+ data translation and processing."""
+
+toolchain = {'name': 'foss', 'version': '2021a'}
+toolchainopts = {'usempi': True}
+
+source_urls = ['https://download.osgeo.org/gdal/%(version)s/']
+sources = [SOURCELOWER_TAR_XZ]
+patches = ['GDAL-3.0.0_fix-python-CC-CXX.patch']
+checksums = [
+    '190c8f4b56afc767f43836b2a5cd53cc52ee7fdc25eb78c6079c5a244e28efa7',  # gdal-3.3.0.tar.xz
+    '223a0ed1afb245527d546bb19e4f80c00f768516ab106d82e53cf36b5a1a2381',  # GDAL-3.0.0_fix-python-CC-CXX.patch
+]
+
```

The failed test by @verdurin might be fixed by adding `pkg-config` as build dependency
```suggestion

builddependencies = [
    ('pkg-config', '0.29.2'),
]

```